release notes/v0.35.0.md

```diff
@@ -0,0 +1,94 @@
+k6 v0.35.0 is here! :tada: It introduces several new features that nicely enhance usage of k6 as well as contains a whole number of fixes and ongoing efforts with refactoring.
+
+In total, we have closed 14 issues and merged 43 PRs. We have also branched out three new xk6 extensions during this release :star:
+
+## New features
+
+### VU tags ([#2172](https://github.com/grafana/k6/pull/2172))
+
+k6 now supports setting tags for VUs as part of the Execution API with an easy key-value interface. Example usage:
```

Comment:
I feel like this doesn't explain that metrics emitted by this VU will be tagged with those tags, and I am not certain if that is as obvious to everybody as it is to us

Reply:
Small clarification won't hurt probably :+1: